### PR TITLE
dnsdist: Prepare ChangeLog and secpoll update for 1.7.0-alpha1

### PR DESCRIPTION
**Pull request by rgacogne**
### Checklist

I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

File: .github/actions/spell-check/patterns.txt [hunk 1]
```diff
@@ -60,3 +60,6 @@ C0FFEE
 # DoT: DNS over TLS is often combined with other words (images, links, ...), same for DoH
 DoT
 DoH
+
+# Contributor with a non-ascii character in their name
+Hoffst[^[:ascii:]]tte
```
Comment (jsoref) [inline .github/actions/spell-check/patterns.txt:65]:
```suggestion
Hoffst..tte
```

Comment (jsoref) [reply .github/actions/spell-check/patterns.txt:65]: If you wanted to be stricter, this would work:
```suggestion
Hoffst[^[:ascii:]]+tte
```

Comment (rgacogne) [reply .github/actions/spell-check/patterns.txt:65]: ah, right, it counts as two characters!

**Comment by rgacogne** [comment]
CI failures are unrelated (includeDirectory(), missing libsodium for ixfrdist, missing promtool for rec).

**Comment by Habbie** [comment]
> missing libsodium for ixfrdist, missing promtool for rec

rebase should fix these two, but I can see how you just don't care here :)

**Pull request merged**